azure-devops/global/abstracts/replace-env-vars.yml

```diff
@@ -0,0 +1,15 @@
+parameters:
+  - name: FILE
+    type: string
+  - name: displayName
```

Comment:
If we are using all parameters in UPPERCASE, we should be consistent and set all of them uppercase.